### ISSUE TITLE
[sf][a11Y][Button]: Button component can has the "button menu" announcement

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents-react#2683]
**Describe the bug**
Hi Colleague,
I am a member of Shanghai WEF team. When we use Button component to open an action sheet, the a11y expect hopes that since this is a button that opens a pop over menu, aria-haspopup="menu" must be present on the "button" element so that the "button menu" announcement is made by screen reader. However, the Button component of ui5-webcomponents-react doesn't offer such ability, could you please give us some advice? Thanks!

**Isolated Example**
Here is an isolated example if possible by forking: https://codesandbox.io/embed/affectionate-lichterman-5ee1o2?fontsize=14&hidenavigation=1&theme=dark

**To Reproduce**
Environment & instance:
https://qacand.hcm.ondemand.com/login?bplte_company=TALCDC12LilyH
admin/pwd

1. Login -> Go to OMP
2. Click on '.Manage my opportunities'
3. Tab to the ellipsis

**Expected behavior**
Since this is a button that opens a pop over menu, aria-haspopup="menu" must be present on the "button" element so that the "button menu" announcement is made by screen reader. This is missing. Refer to this UI5 sample: https://ui5.sap.com/#/entity/sap.m.ActionSheet/sample/sap.m.sample.ActionSheet

**Screenshots**
![button_menu_omp](https://user-images.githubusercontent.com/10113427/158498920-9c85bbad-b045-46b1-b68e-ed9c8019169e.png)

![button_menu_omp1](https://user-images.githubusercontent.com/10113427/158498814-d3d1fb39-df96-4cc7-ad2d-fe2425f6fba6.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.2.1
`@ui5/webcomponents-react` version: 0.22.2
Browser: Chrome

**Comment by Lukas742** [comment]
Hi @tandan0813 

you can use the `accessibilityAttributes` web component property for this.
https://github.com/SAP/ui5-webcomponents-react/blob/3acfeb6a21eaf7e4c2e5cfdf36da28e66fd437db/packages/main/src/webComponents/Button/index.tsx#L57-L73

[codeSandbox](https://codesandbox.io/s/headless-monad-uv6nwx?file=/src/App.js)

**Comment by tandan0813** [comment]
Hi @Lukas742 
Here is another issue. When I add accessibilityAttributes to the Button, Button in the menu list also has this property, which is not in line with expectations. Could you please help take a look at it ?
<img width="987" alt="image" src="https://user-images.githubusercontent.com/10113427/159160621-6a58a3d4-9eee-4e2d-a1d4-eed8f62e84f6.png">

**Comment by Lukas742** [comment]
Hi @tandan0813 

this seems like a bug in the `@ui5/webcomponents` to me. I raised [an issue ](https://github.com/SAP/ui5-webcomponents/issues/4933)so our colleagues there can take over.

**Comment by Lukas742** [comment]
Hi @tandan0813 

the example I posted was incorrectly implemented. I wasn't aware that the `accessibilityAttributes` need to be assigned a new object. (See https://github.com/SAP/ui5-webcomponents/issues/4933#issuecomment-1084368823)

If you implement it like [this](https://codesandbox.io/s/optimistic-chandrasekhar-905u6d?file=/src/App.js) it should be working as expected.
